code/controllers/subsystem/shuttles.dm

```diff
@@ -58,18 +57,39 @@ var/datum/subsystem/shuttle/SSshuttle
 			continue
 		supply_packs[P.type] = P
 
+	for(var/shipping in subtypesof(/datum/shipping))
+		var/datum/shipping/S = new shipping()
+		if(!S.abstract)
+			shipping_datums += S
+
 	initial_move()
 	..()
 
 
 /datum/subsystem/shuttle/fire()
-	points += points_per_decisecond * wait
 	for(var/thing in mobile)
 		if(thing)
 			var/obj/docking_port/mobile/P = thing
 			P.check()
 			continue
 		mobile.Remove(thing)
+	if(prob(5))
+		fluctuate()
+	if(prob(1) && prob(10))
```

Comment:
well I mean you can do this or you can do `prob(0.1)` but it really doesn't matter too much